internal/mock/mock.go

```diff
@@ -17,6 +20,10 @@ func (w *ws) Close() error {
 	return nil
 }
 
+func (w *ws) LiveNotifications(id string) (chan rpc.RPCResponse, error) {
+	return nil, nil
```

Comment:
Instead or `nil, nil` you can return an error. `nil, errors.New("live queries are unimplemented for mocks")`